doc_source/efa-start-nccl-base.md

```diff
@@ -406,10 +406,10 @@ Install NCCL\. For more information about NCCL, see the [NCCL repository](https:
    $ cd /opt
    ```
 
-1. Clone the official NCCL repository to the instance and navigate into the local cloned repository\.
+1. Clone the an [aws-ofi-compatible version of NCCL](https://github.com/aws/aws-ofi-nccl/blob/master/RELEASENOTES.md) from the official NVidia repository to the instance and navigate into the local cloned repository\.
```

Comment:
All the aws-ofi-nccl plugin versions should work with NCCL version > 2.4.8. Please open a ticket to plugin if you find anything otherwise.